### PR TITLE
1467905 - Added error handling for images with improper APB Spec

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: pkg/apb/rhcc_registry.go [hunk 1]
```diff
@@ -61,39 +61,39 @@ func (r RHCCRegistry) LoadSpecs() ([]*Spec, int, error) {
...
+		r.log.Info("Could not form request. Error: %s. Skipping this Image.", err)
+		return nil
 	}
 
 	req.Header.Add("Accept", "application/json")
 
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		return nil, err
+		r.log.Info("Could not send request. Error: %s. Skipping this Image.", err)
```
Comment (shawn-hurley) [inline pkg/apb/rhcc_registry.go]: We could print the image name here for better alerting.

**Comment by eriknelson** [comment]
@dymurray tests are red. Also, what's the release bugfix policy now? This fix is going into master and backported to the release branch?

**Comment by jmrodri** [comment]
@eriknelson this is already in release-0.9, this pr is to bring that change over to master

**Comment by eriknelson** [comment]
@dymurray @jmrodri Ah, got it! TY

**Pull request merged**